starter_code/app2.js

```diff
@@ -0,0 +1,36 @@
+function changeBackground(){
+		
+var city = $('#city-type:text').val();
+
+if ((city === "Sydney")|| (city === "SYD")){
+	$('body').css('backgroundImage','url(images/syd.jpg)');	
+};
+
+else if ((city === "Los Angeles")|| (city === "LA") || (city === "LAX")){
+$('body').css('backgroundImage','url(images/los.jpg)');	
+};		
+
+else if ((city === "New York")|| (city === "New York City") || (city === "NYC")){
+$('body').css('backgroundImage','url(images/nyc.jpg)');	
+};		
+
+else if ((city === "Austin")|| (city === "ATX")){
+$('body').css('backgroundImage','url(images/aus.jpg)');
+};			
+
+else ((city === "San Francisco")|| (city === "SF") || (city === "Bay Area")){
+$('body').css('backgroundImage','url(images/san.jpg)');
+};	
+
+$( document ).ready(function() {
```

Comment:
Document ready is a listener to wait for the entire html to load.  In some pages, it takes a long time for the html to load so we have a document ready listener which will fire off jQuery events after the page has fully loaded.